.github/CONTRIBUTING.md

```diff
@@ -5,23 +5,37 @@ Thank you for considering to help Wagtail.
 We welcome all support, whether on bug reports, code, design, reviews, tests,
 documentation, translations or just feature requests.
 
-## Using the issue tracker
+## Working on an issue
```

Comment:
The diff here looks bigger than it is, because I moved the "New code" section to the top and re-titled it.